CIScripts/Test/PesterLogger/PesterLogger.Tests.ps1

```diff
@@ -0,0 +1,58 @@
+﻿$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$sut = (Split-Path -Leaf $MyInvocation.MyCommand.Path) -replace '\.Tests\.', '.'
+. "$here\$sut"
+
+Describe "PesterLogger" {
+    function InMemoryAddContent() {
+        Param($Path, $Value)
+        $Script:Content += $Value
+        $Script:Path = $Path
+    }
+
+    function MakeFakeWriter {
+        $Script:Content = ""
+        $Script:Path = ""
+        Get-Item function:InMemoryAddContent
+    }
+
+    Context "New-PesterLogger" {
+        It "registers a new global Write-Log function" {
+            New-PesterLogger -OutDir "some_dir"
```

Comment:
I think that the `Set` or `Register` verb would be more appropriate here, as `New` usually the created resource, which is not a case here.

Reply:
Gonna go with Initialize